hackvote/app/views/events/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<% provide(:title, @event.title) %>
+
+
+<h1><%= @event.title %></h1>
+<div>
+  <h2>Add new Project</h2>
+  <%= render 'shared/project_form' %>
+  <%= button_to "Close Voting", close_event_path, params: { id: @event.id }, confirm: "are you sure?" %>
+</div>
+<div>
+  <%= session[@event.id] %> votes remaining
+</div>
+
+<div>
+  <h2>Projects</h2>
+  <% @projects.each do |project| %>
+    <% if not(project.name.nil? or project.creators.nil?) %>
```

Comment:
I forget exactly the full details, but `or` has a different precedence than `||` (same with `and` and `&&`). Style guide suggests never using `or` https://github.com/Heyzap/heyzap.com/wiki/Ruby-Styleguide